app/models/study.rb

```diff
@@ -1196,6 +1196,33 @@ def default_color_profile
     self.default_options[:color_profile].presence
   end
 
+  # array of names of annotations to ignore the unique values limit for visualizing
+  def override_viz_limit_annotations
+    self.default_options[:override_viz_limit_annotations] || []
+  end
+
+  # make an annotation visualizable despite exceeding the default values limit
+  def add_override_viz_limit_annotation(annotation_name)
+    cell_metadatum = self.cell_metadata.find_by(name: annotation_name)
+    if cell_metadatum
+      # we need to populate the 'values' array, since that will not have been done at ingest
+      begin
+        uniq_vals = cell_metadatum.concatenate_data_arrays(annotation_name, 'annotations').uniq
+        cell_metadatum.update!(values: uniq_vals)
+      rescue => e
+        Rails.logger.error "Could not cache unique annotation values: #{e.message}"
```

Comment:
Non-blocking: since this will be run from the Rails console, it would be more helpful to log these with `puts` so that the user can see the error.